server_plugin/server.py

```diff
@@ -254,7 +272,7 @@ def _get_state(vca_client):
         for connection in nw_connections}
 
     for connection in nw_connections:
-        if connection['network_name'] == management_network_name:
+        if ctx.node.properties.get('use_external_resource') or connection['network_name'] == management_network_name:
```

Comment:
I already planned to remove this change based on the exchange I had with @0lvin . I had originally thought I'd be able to allow the management network to be omitted for external resources and would use the (admittedly questionable) logic I introduced here. I now see that was a mistake and that the specification of the management network will be mandatory for external resources.